components/autocomplete/overview.md

```diff
@@ -110,15 +110,23 @@ You can find more options for customizing the AutoComplete styling in the [Appea
 
 ### Popup settings
 
-The popup of the component can be additionally customized via nested tags:
+The AutoComplete exposes settings for its dropdown (popup). To configure the options, declare an  `<AutoCompletePopupSettings>` tag inside the `<AutoCompleteSettings>` tag:
 
-<div class="skip-repl"></div>
-````
-<TelerikAutoComplete>
+````CSHTML
+<TelerikAutoComplete Data="@Data"
```

Comment:
Ideally, this example should enable the user to change the number of visible items in the dropdown. This will show how all the popup settings work together. For example, use more items and enable filtering. If needed, generate random data with a loop.